downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc

```diff
@@ -0,0 +1,21 @@
+[id="con-aap-azure-infrastructure-usage"]
+
+= {AAPonAzureNameShort} Infrastructure usage
+
+When you install {AAPonAzureNameShort}, the following infrastructure is deployed into your Azure subscription:
```

Comment:
Do you mean environment? 
`into your Azure subscription:` > `into your Azure environment:`

Reply:
Disregard that note, subscription is correct.